packages/admin/cms-admin/src/dam/DataGrid/selection/DamMoreActions.tsx

```diff
@@ -114,7 +114,14 @@ export const DamMoreActions = ({ button, transformOrigin, anchorOrigin, folderId
 
     return (
         <>
-            {React.cloneElement(button, { onClick: handleClick })}
+            {React.cloneElement(
+                button,
+                { onClick: handleClick },
+                <>
+                    {button.props.children}
+                    {lengthOfSelectedFiles > 0 && <NumberSelectedChip>{selectionSize}</NumberSelectedChip>}
+                </>,
+            )}
```

Comment:
How does this work in SelectionFooter? It has no `children` to override

Reply:
The chip will not display in that case, but i have added an additional check to make sure the a child was provided to the button 
![image](https://github.com/vivid-planet/comet/assets/72387685/2d77cc30-aba0-4719-ae25-11b2f8364209)

Reply:
I'd prefer that instead of using it in two places and making sure that it's not shown in one, just add it to the other place. This would be in the DamMoreActions in `DamTable`:

https://github.com/vivid-planet/comet/blob/e146d8bb9012d91b70481fa92f2ab863f52aa295/packages/admin/cms-admin/src/dam/DamTable.tsx#L88-L92

Reply:
But wouldn´t that produce a bit of duplicate code that is needed to calculate the amount of selected items ?

Reply:
It's okay to have duplicate code for a better solution 🙂

Reply:
i moved this part outside of the component